package.json

```diff
@@ -38,6 +37,7 @@
     "@fortawesome/free-regular-svg-icons": "6.5.2",
     "@fortawesome/free-solid-svg-icons": "6.5.2",
     "@fortawesome/react-fontawesome": "0.2.0",
+    "@openedx/frontend-slot-footer": "^1.0.2",
```

Comment:
Shouldn't we add an explicit FPF dependency here, as in the other MFEs?

Reply:
It's a straight dependency in `frontend-slot-footer` https://github.com/openedx/frontend-slot-footer/blob/53987a478f615ec775137cc1bdfba982b8d0b3b6/package.json#L35 so I'm fine either way. If you'd prefer we have FPF explicitly set as a dependency in all the MFEs using `frontend-slot-footer` I'm more than happy to make that change.

Reply:
Yeah, I guess adding a dependency if it's not used directly in the code is probably more confusing than otherwise.  Let's leave it out.

Reply:
agreed on leaving it out until we specifically need it for Profile and it reduces the number of dependencies to update for profile